src/content/changes/highlights/2022-08-05.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2022-08-05T10:00:00
+title: Highlights for the week ending August 5, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+
+## Apps
+
```

Comment:
```suggestion
- [kong-app](https://github.com/giantswarm/kong-app) version [v2.11.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#2110---2022-08-03) Align with upstream chart version [2.11.0](https://github.com/Kong/charts/releases/tag/kong-2.11.0) ([Changes in upstream repository](https://github.com/Kong/charts/compare/kong-2.8.2...kong-2.11.0)) Update kong/kubernetes-ingress-controller to [2.5.0](https://github.com/Kong/kubernetes-ingress-controller/releases/tag/v2.5.0)
- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) version [v2.15.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2150---2022-08-03) added Support for annotations, labels and suffix on the internal controller service.
NOTE: Adding, changing or removing the suffix results in a different name of the controller service resource. Since Helm does not keep track of the old resource, we recommend to uninstall and reinstall the app when changing the suffix. Changed: Omit service.beta.kubernetes.io/aws-load-balancer-proxy-protocol for use-proxy-protocol: "false"
-[linkerd2-app](https://github.com/giantswarm/linkerd2-app) version [v0.7.2](https://github.com/giantswarm/linkerd2-app/blob/master/CHANGELOG.md#072---2022-08-03) Add Giant Swarm team label to resources and init container to destination and injector services to avoid the [known issue](https://github.com/projectcalico/calico/issues/3499) of missing IP in post hook action. Update pytest-helm-charts from beta to v0.7.0 ([#84](https://github.com/giantswarm/linkerd2-app/pull/84))
-[linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) version [v0.7.2](https://github.com/giantswarm/linkerd2-cni-app/blob/master/CHANGELOG.md#072---2022-08-03) Add Giant Swarm team label to resources and Update pytest-helm-charts from beta to [v0.7.0](https://github.com/giantswarm/pytest-helm-charts/blob/master/CHANGELOG.md) ([#68](https://github.com/giantswarm/linkerd2-cni-app/pull/68))

```